typings/wavesurfer-defs.d.ts

```diff
@@ -0,0 +1,73 @@
+// Code - https://raw.githubusercontent.com/katspaugh/wavesurfer.js/master/src/wavesurfer.js
```

Comment:
Also, why are we using the master branch file here? Should we use this file from v3.3.3?

Reply:
Oh, yeah correct. Done

Reply:
Have you tested the type.defs with the file in v3.3.3 (as it can be different from master branch)?

Reply:
Yup! Did that just now.